public/js/practise.js

```diff
@@ -88,7 +90,7 @@ console.log(result);
 
 // Ex 7 numarul de consoane dintru-un cuvant cu functie si parametru //
```

Comment:
This exercise has the similar aproach like the **Ex 6. Numarul de Vocale**, please think of a **SOLUTION** without using **promt()** function from **JS**.
You can simply pass in a **string** or a **variable** like this:

`let string = 'AnyWordYouWant'`
`let result = countConsonant(string)`
`console.log(result);`